coalib/bearlib/aspects/root.py

```diff
@@ -0,0 +1,110 @@
+from .base import aspectbase
+from .meta import aspectclass
+
+
+class Root(aspectbase, metaclass=aspectclass):
```

Comment:
Why this file named `root.py`, not `Root.py` like other aspect-tree module? IMO it's better to be consistent in here.